index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE HTML>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Compotes</title>
+        <script src="js/jquery-3.1.0.min.js"></script>
```

Comment:
Wouldn't it work better to just use the absolute addressing: `/js/jquery/...` ?

Also, if it's possible, please delete bootstrap and jquery directories and move the appropriate files into either `js/` or `css/` directories as necessary. Thanks.